src/templates/search/index.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello BaseController!{% endblock %}
+
+{% block body %}
+    <nav>
+        {{ form_start(searchForm) }}
+        {{ form_widget(searchForm) }}
+        {{ form_end(searchForm) }}
+    </nav>
+{% endblock %}
```

Comment:
I think searchbar just need to be a partials included in some pages, but pages shouldn't be based on search